### ISSUE TITLE
Error when building sample-Groceries with typescript@2.5.3

### ISSUE THREAD

**Issue by EwelinaLuiza** [NativeScript/nativescript-dev-webpack#313]
I'm trying to build https://github.com/NativeScript/sample-Groceries with typescript@2.5.3.

Output:
```
ERROR in Error encountered resolving symbol values statically. Could not resolve tns-core-modules/ui/frame relative to /home/ewelina/praca/workspace-native/grocery-2/sample-Groceries/node_modules/nativescript-angular/platform-providers.d.ts., resolving symbol defaultFrameProvider in /home/ewelina/praca/workspace-native/grocery-2/sample-Groceries/node_modules/nativescript-angular/platform-providers.d.ts, resolving symbol NativeScriptCommonModule in /home/ewelina/praca/workspace-native/grocery-2/sample-Groceries/node_modules/nativescript-angular/common.d.ts, resolving symbol NativeScriptCommonModule in /home/ewelina/praca/workspace-native/grocery-2/sample-Groceries/node_modules/nativescript-angular/common.d.ts

ERROR in ./main.aot.ts
Module not found: Error: Can't resolve './app.module.ngfactory' in '/home/ewelina/praca/workspace-native/grocery-2/sample-Groceries/app'
 @ ./main.aot.ts 5:29-62
```
This version works fine:
`    "typescript": "~2.4.2",`

My versions:
```
┌──────────────────┬─────────────────┬────────────────┬──────────────────┐
│ Component        │ Current version │ Latest version │ Information      │
│ nativescript     │ 3.3.0           │  3.3.0         │ Up to date       │
│ tns-core-modules │ 3.3.0           │  3.3.0         │ Up to date       │
│ tns-android      │ 3.3.0           │  3.3.0         │ Up to date       │
│ tns-ios          │ 3.3.0           │  3.3.0         │ Up to date       │
└──────────────────┴─────────────────┴────────────────┴──────────────────┘
```

**Comment by NickIliev** [comment]
@EwelinaLuiza indeed there are some incompatibilities with TS 2.5.x  which we are currently investigating.
As you yourself have noticed reverting the TS to 2.4.x is resolving the issue at this very moment.

**Comment by sis0k0** [comment]
Fixed with {N} 3.4, Angular 5 and nativescript-dev-webpack 0.9.0+.
